vega_sim/scenario/common/agents.py

```diff
@@ -1125,6 +1129,90 @@ def step(self, vega_state: VegaState):
                 key_name=self.key_name,
             )
 
+    def _scale_orders(self, buy_shape: List[MMOrder], sell_shape: List[MMOrder]):
+
+        est_midprice = 0.5 * (buy_shape[0].price + sell_shape[0].price)
```

Comment:
I know it's only used to estimate liquidity but you shouldn't need mid. You can always use just the offset. The probability of trading is (by our definition) 1/2 between best bid and ask and then decays using the prob of trading function.

Reply:
Ah nice thanks, that makes sense, I'll make that change